src_assets/common/assets/web/configs/tabs/Advanced.vue

```diff
@@ -80,7 +80,7 @@ const config = ref(props.config)
     </div>
 
     <!-- Encoder -->
-    <div class="mb-3">
+    <div>
```

Comment:
This will be the only section on this page without `mb-3`?

I think you probably did this because it's the last one, but I fear it will get replicated because new sections are often copied and pasted.

Comment applies on many of the .vue pages.

Reply:
Ok, will undo this.